modules/041-linstor/templates/linstor-affinity-controller/deployment.yaml

```diff
@@ -63,7 +63,7 @@ spec:
     spec:
       {{- include "helm_lib_priority_class" (tuple . "system-cluster-critical") | nindent 6 }}
       {{- include "helm_lib_node_selector" (tuple . "system") | nindent 6 }}
-      {{- include "helm_lib_tolerations" (tuple . "system") | nindent 6 }}
+      {{- include "helm_lib_tolerations" (tuple . "system" (dict) true) | nindent 6 }}
```

Comment:
We don't use the "true" arg anywhere else. Consider using `any-uninitialized-node` mb?

Reply:
Okay I added additional decorator `helm_lib_tolerations_with_drbd_problems`, this should tolerate exactly DRBD problems

Reply:
It is better to add a strategy to helm_lib_tolerations.